lib/api/api.js

```diff
@@ -82,6 +84,9 @@ const api = {
                     }
                 }
             }
+            // issue 100 Continue to the client
+            writeContinue(request, response, apiMethod);
+
```

Comment:
I think you need a writeContinue call at around line 50 for the apiMethods that don't require a check with vault.